docs/docs/content/migration/01_start_migration.en.md

```diff
@@ -104,35 +108,38 @@ export default {
     isSaving: false,
     error: null
   },
+  modules: {
+    subModule: SubModule()
+  },
   signals: {
     newItemTitleSubmitted: submitNewItemTitle
   }
 }
 ```
 
 ### Operators
```

Comment:
Yes, this was just my first pass. Will also need to update again when computed functions are changed.